### PR TITLE
Update the index.html, style.css, and add images folder. Also, change…

### PR DESCRIPTION
**Pull request by Daljin**
… appengine.xml

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,166 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body{
+    overflow-x: hidden;
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css]: nit: Please add a space before the { for all the rules.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,166 @@
...
+}
+
+#intro p{
+    margin: 0;
+    line-height: 1.3;
+}
+
+.aboutContent{
+    width: 50%;
+  	padding-right: 15px;
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css]: Fix formatting here and also order the properties alphabetic order.

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (Daljin) [reply portfolio/src/main/webapp/style.css]: The change has been made.

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (Daljin) [reply portfolio/src/main/webapp/style.css]: I've finished ordering the properties and the rules as you requested.

**Pull request merged**